### PR TITLE
Add rendering test for "select all" checkbox

### PR DESCRIPTION
**Pull request by SirCmpwn**

### CODE REVIEW

File: test/linodes/layouts/IndexPage.spec.js [hunk 1]
```diff
@@ -105,7 +105,6 @@ describe('linodes/layouts/IndexPage', () => {
 
   function makePowerTest(dropdown, endpoint) {
     return async () => {
-      const dispatch = sinon.spy();
```
Comment (eatonphil) [inline test/linodes/layouts/IndexPage.spec.js]: Thanks

**Comment by eatonphil** [comment]
Would really like to see more tests here. Is the checkbox cloud-scale?

**Comment by SirCmpwn** [comment]
There's another test stub for verifying its behavior. Not really sure what to test beyond rendering in this test.

**Comment by eatonphil** [comment]
Just screwing around. Looks fine. Ready for merge?

**Comment by SirCmpwn** [comment]
Yep.

**Comment by SirCmpwn** [comment]
Thanks!

**Pull request merged**